test/lint.js

```diff
@@ -139,7 +143,7 @@ const hasErrors = argv.files
 
 if (hasErrors) {
   console.warn('');
-  console.warn(chalk`{red Problems in }{red.bold ${filesWithErrors.size}}{red  ${filesWithErrors.size === 1 ? 'file' : 'files'}:}`,
+  console.warn(chalk`{red Problems in {bold ${filesWithErrors.size}} ${filesWithErrors.size === 1 ? 'file' : 'files'}:}`,
```

Comment:
This is the same problem mentioned in my comment here: https://github.com/mdn/browser-compat-data/pull/4264#discussion_r291740183

Reply:
I’m pulling this out into a new PR with my reasoning behind it.

Reply:
Thank you, appreciate it!  (I'm all for this change since it looks a lot cleaner, for the record -- I just think it should be in its own PR.  :wink:)